source/content/guides/git/05-undo-commits.md

```diff
@@ -183,6 +188,30 @@ You can reset your Dev environment history to match the current state of your Li
     git push origin master -f
     ```
 
+</Tab>
+
+<Tab title="Integrated Composer" id="integrated-composer">
+
+<Alert title="Note" type="info">
+
+We've adjusted the following steps for [Integrated Composer sites](/guides/integrated-composer), so that you reset history to the **second** to last commit hash on the Live environment, rather than the first - to avoid resetting dev's history to a build artifact.
+
+</Alert>
+
+1. Identify the **second** most recent commit deployed to Live.
+
+1. Run the command below to overwrite the history on Dev's codebase to reflect Live (replace `<site>` with your site's name):
+
+    ```bash{promptUser: user}
+    git reset --hard `terminus env:code-log <site>.live --format=string | grep -m2 'live' | tail -n 1 | cut -f 4`
```

Comment:
Nice! Yes, this command works as I would expect, it gets the second newest commit, and resets the local. So that's good!